docker-compose/bakers.yaml

```diff
@@ -2,67 +2,56 @@ version: '3'
 services:
   bootstrapper:
     image: concordium/dev-node:latest
-    depends_on:
-    - baker_id_gen
-    - collector_backend
+    entrypoint: /bootstrapper-entrypoint.sh
     environment:
     - RUST_BACKTRACE=1
     - RUST_LOG=info
-    - MODE=local_bootstrapper
-    - NUM_BAKERS=${NUM_BAKERS}
-    - EXTRA_ARGS=${EXTRA_ARGS}
-    - DATA_DIR=/var/lib/concordium/data
-    - CONFIG_DIR=/var/lib/concordium/config
-    entrypoint:
-    - /start.sh
+    - CONCORDIUM_NODE_DATA_DIR=/var/lib/concordium/data
+    - CONCORDIUM_NODE_CONFIG_DIR=/var/lib/concordium/config
+    - CONCORDIUM_NODE_BOOTSTRAPPER_REGENESIS_BLOCK_HASHES_FILE=/genesis-data/genesis-${NUM_BAKERS}-bakers/genesis_hash
   collector_backend:
     image: concordium/dev-node:latest
+    entrypoint: /node-collector-backend
     environment:
     - RUST_BACKTRACE=1
     - RUST_LOG=info
-    - MODE=local_collector_backend
+    - COLLECTOR_BACKEND_ADDRESS=0.0.0.0
     - COLLECTOR_BACKEND_PORT=10000
-    - COLLECTOR_BACKEND_HOST=0.0.0.0
-    - EXTRA_ARGS=${EXTRA_ARGS}
-    entrypoint:
-    - /start.sh
     ports:
-    - 12000:10000
+    - "12000:10000"
   baker_id_gen:
     image: concordium/dev-node:latest
-    entrypoint:
-    - /baker_id_generator
+    entrypoint: /baker_id_generator
   baker:
     image: concordium/dev-node:latest
     depends_on:
     - bootstrapper
-    entrypoint:
-    - /start.sh
+    - baker_id_gen
+    entrypoint: /node-entrypoint.sh
     environment:
     - RUST_BACKTRACE=1
     - RUST_LOG=info
-    - DESIRED_PEERS=${DESIRED_PEERS}
-    - BOOTSTRAP_FIRST_NODE=bootstrapper:8888
-    - EXTERNAL_PORT=8888
-    - NUM_BAKERS=${NUM_BAKERS}
-    - MODE=local_basic
-    - DATA_DIR=/var/lib/concordium/data
-    - CONFIG_DIR=/var/lib/concordium/config
-    - RPC_SERVER_ADDR=0.0.0.0
-    - EXTRA_ARGS=${EXTRA_ARGS}
+    - LD_LIBRARY_PATH=/usr/local/lib
```

Comment:
I don't think this is necessary since the node is linked statically.

Reply:
done